src/main/java/co/rsk/bitcoinj/script/FlyoverRedeemScriptParser.java

```diff
@@ -0,0 +1,114 @@
+package co.rsk.bitcoinj.script;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import co.rsk.bitcoinj.core.BtcECKey;
+import co.rsk.bitcoinj.core.Sha256Hash;
+import co.rsk.bitcoinj.core.Utils;
+import co.rsk.bitcoinj.core.VerificationException;
+import co.rsk.bitcoinj.crypto.TransactionSignature;
+import com.google.common.collect.Lists;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class FlyoverRedeemScriptParser implements RedeemScriptParser {
+
+    private static final Logger logger = LoggerFactory.getLogger(FlyoverRedeemScriptParser.class);
+
+    private final MultiSigType multiSigType;
+    private final List<ScriptChunk> redeemScriptChunks;
+    private final RedeemScriptParser redeemScriptParser;
+
+    public FlyoverRedeemScriptParser(List<ScriptChunk> redeemScriptChunks) {
+        this.redeemScriptChunks = extractStandardRedeemScriptChunks(redeemScriptChunks);
+        this.redeemScriptParser = RedeemScriptParserFactory.get(redeemScriptChunks);
+        this.multiSigType = MultiSigType.FLYOVER;
+    }
+
+    @Override
+    public ScriptType getScriptType() {
```

Comment:
This method will be removed from the `RedeemScriptParser` interface.

Reply:
Rebase your branch